simulation/executor/internal/executortypes/simmanager.go

```diff
@@ -92,18 +93,21 @@ func (m Manager) legacyActions(seed int64, cdc codec.JSONCodec) []simtypes.Actio
 	// hardcode this one filepath for wasm.
 	// TODO: Clean this up / make a better plan
 	simState := module.SimulationState{
-		AppParams:    loadAppParamsForWasm("params.json"),
-		ParamChanges: []simulation.ParamChange{},
-		Contents:     []simulation.WeightedProposalContent{},
-		Cdc:          cdc,
+		AppParams:              loadAppParamsForWasm("params.json"),
+		LegacyParamChange:      []simulation.LegacyParamChange{},
+		LegacyProposalContents: []simulation.WeightedProposalContent{},
+		Cdc:                    cdc,
 	}
 
 	r := rand.New(rand.NewSource(seed))
 	// first pass generate randomized params + proposal contents
 	for _, moduleName := range m.moduleManager.OrderInitGenesis {
 		if simModule, ok := m.legacyModules[moduleName]; ok {
-			simState.ParamChanges = append(simState.ParamChanges, simModule.RandomizedParams(r)...)
-			simState.Contents = append(simState.Contents, simModule.ProposalContents(simState)...)
+			// UNFORKINGTODO OQ: Figure out how to call RandomizedParams and ProposalContents, I don't understand
+			// Printing so we dont need to uncomment more, just delete the prints later
+			// simState.LegacyParamChange = append(simState.LegacyParamChange, simModule.RandomizedParams(r)...)
+			// simState.LegacyProposalContents = append(simState.LegacyProposalContents, simModule.ProposalContents(simState)...)
```

Comment:
I think this does not matter for now. We can just keep this commented out and deal with it later or potentially never. If things are moving off of param code, we can just only do fuzzing on new style things.

Reply:
Changed to unforkingnote https://github.com/osmosis-labs/osmosis/pull/6413/commits/2e1c779ac8cb2bf84cc0b1f1c9d0faaffc7d30aa